package.json

```diff
@@ -5,9 +5,27 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "files": [
+    "/bin",
     "/lib"
   ],
+  "bin": {
+    "strong-config": "./bin/run"
+  },
+  "oclif": {
+    "commands": "./lib/cli/commands",
+    "bin": "strong-config",
+    "plugins": [
+      "@oclif/plugin-help",
+      "@oclif/plugin-warn-if-update-available",
+      "@oclif/plugin-not-found"
```

Comment:
Could the [autocomplete plugin](https://github.com/oclif/plugin-autocomplete) be useful to us?

Reply:
Thanks for the pointer. Will look into it. (resolving the duplicate comment)

Reply:
cool, will leave this open until you made a decision on whether we should use it or not use it